components/motor/tmcstepper/stepper_motor_tmc.go

```diff
@@ -25,21 +25,22 @@ type PinConfig struct {
 }
 
 // TMC5072Config describes the configuration of a motor.
+//nolint:lll
 type TMC5072Config struct {
 	Pins             PinConfig `json:"pins"`
-	BoardName        string    `json:"board"`                      // used to get encoders
-	MaxRPM           float64   `json:"max_rpm,omitempty"`          // RPM
-	MaxAcceleration  float64   `json:"max_acceleration,omitempty"` // RPM per second
+	BoardName        string    `json:"board"` // used to get encoders
+	MaxRPM           float64   `json:"max_rpm,omitempty"`
+	MaxAcceleration  float64   `json:"max_acceleration_rpm_per_sec,omitempty"`
 	TicksPerRotation int       `json:"ticks_per_rotation"`
 	SPIBus           string    `json:"spi_bus"`
 	ChipSelect       string    `json:"chip_select"`
 	Index            int       `json:"index"`
-	SGThresh         int32     `json:"sg_thresh,omitempty"`
+	SGThresh         int32     `json:"sg_thresh,omitempty"` // TODO(RSDK-568): units?
 	HomeRPM          float64   `json:"home_rpm,omitempty"`
 	CalFactor        float64   `json:"cal_factor,omitempty"`
-	RunCurrent       int32     `json:"run_current,omitempty"`  // 1-32 as a percentage of rsense voltage, 15 default
-	HoldCurrent      int32     `json:"hold_current,omitempty"` // 1-32 as a percentage of rsense voltage, 8 default
-	HoldDelay        int32     `json:"hold_delay,omitempty"`   // 0=instant powerdown, 1-15=delay * 2^18 clocks, 6 default
+	RunCurrent       int32     `json:"run_current,omitempty"`  // 1-32 as a percentage of rsense voltage, 15 default  // TODO(RSDK-568): units?
```

Comment:
what's the data sheet?

Reply:
This garbage https://www.trinamic.com/fileadmin/assets/Products/ICs_Documents/TMC2130_datasheet.pdf

Reply:
gotcha - for these I think we should keep the field names unit-less since they aren't really standard (e.g. `factor / 32`, `clocks per current decrement`) and make sure our docs reflect the comments we have in code